pom.xml

```diff
@@ -79,12 +79,27 @@
       <artifactId>xmlresolver</artifactId>
       <version>4.5.1</version>
     </dependency>
+    <dependency>
```

Comment:
Why did you add this dependency ?

Reply:
Because according to running `dependency:analyze-only` it is in use in the code base, which I suspect is correct ;-)

Reply:
It is.

but `org.eclipse.sisu:org.eclipse.sisu.plexus` is included by `org.apache.maven:maven-plugin-api` and by `org.apache.maven:maven-core`, and I have dependencies declared on this ones. So why do you add a direct declaration to this dependency, which is transitively included ? To explicitely declare that I use this dependency ?

Reply:
I'm not entirely sure... Transitive dependencies are very tricky. But I tend to find it you make the dependency plugin happy then you usually have less problems. Feel free to adjust my PR if you prefer; you should be able to push commits to my branch if you want to make some changes.